### PR TITLE
chore(comp): Remove unnecessary completion code

### PR DESCRIPTION
**Pull request by marckhouzam**
**What this PR does / why we need it**:

Cleanup of unnecessary code after moving to Cobra 1.1.1

With the move to Cobra 1.1.1, all commands that have sub-commands have file completion automatically disabled, so helm need not do it itself.

### CODE REVIEW

**Review by mattfarina — APPROVED** [review]
lgtm

**Comment by bacongobbler** [comment]
Quick FYI that you may merge when ready. PRs from core maintainers only require one additional approval from another core maintainer before merging :)

https://github.com/helm/helm/blob/master/CONTRIBUTING.md#size-labels

**Comment by marckhouzam** [comment]
> Quick FYI that you may merge when ready. PRs from core maintainers only require one additional approval from another core maintainer before merging :)
> 
> https://github.com/helm/helm/blob/master/CONTRIBUTING.md#size-labels

Thanks for catching this @bacongobbler.  You are right that I thought I needed a second reviewer.  I had misunderstood the recent change in policy and thought the maintainer author of the PR did not count as a reviewer.

Your diligence is always helpful!

**Pull request merged**
